src/content_cao.cpp

```diff
@@ -1644,6 +1653,8 @@ void GenericCAO::processMessage(const std::string &data)
 				m_animation_range = v2s32((s32)range.X, (s32)range.Y);
 				m_animation_speed = readF1000(is);
 				m_animation_blend = readF1000(is);
+				// these are sent inverted so we get true when the server sends nothing
+				m_animation_loop = !readU8(is);
```

Comment:
I'd prefer if:
  * This was a more generic flags field.
  * This wasn't inverted, but used a length check instead.